integration-tests/utils/integration-helpers.js

```diff
@@ -141,87 +140,17 @@ const withUser = test => async args => {
   }
 }
 
-const addUserToBilling = _.flow(withSignedInPage, withUserToken)(async ({ page, billingProject, email }) => {
-  await page.evaluate((email, billingProject) => {
-    return window.Ajax().Billing.addProjectUser(billingProject, ['User'], email)
-  }, email, billingProject)
-
-  console.info(`added user to: ${billingProject}`)
-
-  const userList = await page.evaluate(billingProject => {
-    return window.Ajax().Billing.listProjectUsers(billingProject)
-  }, billingProject)
-
-  const billingUser = _.find({ email }, userList)
-
-  console.info(`test user was added to the billing project with the role: ${!!billingUser && billingUser.role}`)
-})
-
-const removeUserFromBilling = _.flow(withSignedInPage, withUserToken)(async ({ page, billingProject, email }) => {
-  await page.evaluate((email, billingProject) => {
-    return window.Ajax().Billing.removeProjectUser(billingProject, ['User'], email)
-  }, email, billingProject)
-
-  console.info(`removed user from: ${billingProject}`)
-})
-
-const withBilling = test => async options => {
-  console.log('withBilling ...')
-  await addUserToBilling(options)
-
-  try {
-    await test({ ...options })
-  } finally {
-    console.log('withBilling cleanup ...')
-    await deleteRuntimes(options)
-    await removeUserFromBilling(options)
-  }
-}
-
-const deleteRuntimes = _.flow(withSignedInPage, withUserToken)(async ({ page, billingProject, email }) => {
-  const deletedRuntimes = await page.evaluate(async billingProject => {
-    const runtimes = await window.Ajax().Runtimes.list({ googleProject: billingProject, role: 'creator' })
```

Comment:
Not sure if we actually need `deleteRuntimes`, since we delete the workspace after the test runs anyway. But it was part of `withBilling`, so I'm preserving it here.

It was broken anyway, since it was passing a _billing_ project as a _Google_ project here. This fixes it.

Reply:
I do think think it's important to delete runtimes before deleting the workspace-- otherwise deleting the workspace may fail.

Reply:
deleteRuntime is async...so if it really affects workspace deletion, then some polling to make sure the runtime is indeed deleted will be needed

Reply:
If a workspace fails to delete, it's not a huge problem because we do have a janitor service that deletes old workspaces.

The issue I'm thinking of specifically is related to Galaxy apps-- see https://broadworkbench.atlassian.net/browse/WOR-595 (done, which means the workspace deletion will fail if a Galaxy app is still running) and https://broadworkbench.atlassian.net/browse/IA-3882 (still open).

Reply:
There was no polling for runtime deletion before, so this does not change that aspect.

These tests start Jupyter and RStudio apps. I don't think there's any existing Terra UI test that starts a Galaxy app.